apps/matrix-io/.gitignore

```diff
@@ -0,0 +1,164 @@
+##########################
```

Comment:
@pascalwhoop moved the fabricator to it's own library, so I think it makes slightly more sense to move it into the respective lib.

Reply:
I interpret the libs folder as a location for shared/common code.  matrix-cli is intended to be a standalone CLI application and none of the code here is shared....meaning that no other project depends on it.